Thrive.csproj

```diff
@@ -253,14 +254,11 @@
     <Folder Include="src\auto-evo\simulation\" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="Newtonsoft.Json">
-      <Version>12.0.3</Version>
+    <PackageReference Include="Newtonsoft.Json" Version="12.0.3">
     </PackageReference>
-    <PackageReference Include="System.Numerics.Vectors">
-      <Version>4.5.0</Version>
+    <PackageReference Include="System.Numerics.Vectors" Version="4.5.0">
     </PackageReference>
-    <PackageReference Include="SharpZipLib">
-      <Version>1.3.0</Version>
+    <PackageReference Include="SharpZipLib" Version="1.3.0">
```

Comment:
Please revert these changes, as this separate `Version` xml node version has seriously messed with package installs on my computer.